src/barkhaPJun20/VerifyMobileNumber.java

```diff
@@ -0,0 +1,77 @@
+package barkhaPJun20;
+
+public class VerifyMobileNumber {
+	
+	void validMobileNumber(String num) {
+		//Exa.+91 9765463742
+		if (num.length()==14) {
```

Comment:
Here, you can write both if conditions together by using && operator.

Reply:
Changes Done